contracts/ChainmonstersRewards.cdc

```diff
@@ -16,6 +16,8 @@ pub contract ChainmonstersRewards: NonFungibleToken {
     pub event NFTMinted(NFTID: UInt64, rewardID: UInt32, serialNumber: UInt32)
     pub event NewSeasonStarted(newCurrentSeason: UInt32)
 
+    pub event ItemConsumed(itemID: UInt64, playerId: String, playerAddress: Address)
```

Comment:
`playerAddress` is not available in the context of a consume and should be removed. It can be retreived by reading the authorizers of the transaction in which the event happened